docs/running.rst

```diff
@@ -5,39 +5,164 @@ In this section, we assume everything to be running in an
 environment where "pip install ncolony" has taken place.
 This will usually be a virtualenv, which should be activated.
 
-Running ncolony monitor
------------------------
+Example
```

Comment:
Why are more narrative docs part of a test clean up PR?

Reply:
removed